Code/MinimalLib/minilib.h

```diff
@@ -18,7 +18,10 @@ class JSMol {
   std::string get_smiles() const;
   std::string get_molblock() const;
   std::string get_inchi() const;
-  std::string get_svg() const;
+  std::string get_svg(unsigned int width, unsigned int height) const;
```

Comment:
Is there a reason not to use default args here?

```
std::string get_svg(unsigned int width=250, unsigned int height=200) const;
```

Reply:
Yeah, that doesn’t work well with embind

Reply:
Yeah, that doesn’t work well with embind